src/models/Genre/getAll.ts

```diff
@@ -4,10 +4,21 @@ import { cdaClient, DEFAULT_CONTENTFUL_LIMIT } from '../../utils/contentful';
 import { Genre } from './GenreModel';
 import parseGenre from './parseGenre';
 
+type GetAllOptions = PaginationOptions & {
+	include?: 0 | 1 | 2 | 3 | 4 | 5 | 6 | 7 | 8 | 9 | 10;
```

Comment:
I gave Jesus feedback on this regarding not making this generic values like this. Let's map this to specific strings to make the API interface clearer. Easiest way to accomplish this is to make this something like `include?: 'movies' | 'moviesWithGenres'` and then have a function that remaps to the int value to pass to contentful. Makes for a better implementation for readability.

Reply:
Yes this is what I've got so far!

```ts
	type GetAllOptions = PaginationOptions & {
	include?: 'movies' | 'moviesWithGenre';
	query?: {
		id?: string;
		title?: string;
	};
};

function getIncludeValue(
	include: GetAllOptions['include']
): 0 | 1 | 2 | 3 | 4 | 5 | 6 | 7 | 8 | 9 | 10 {
	switch (include) {
		case 'movies':
			return 2;
		case 'moviesWithGenre':
			return 3;
		default:
			return 0;
	}
}

```

Reply:
What if we use enums? I think we wouldn't need a translator function then:
```typescript
export enum ContentfulIncludeOptions  {
	movies  = 2,
	moviesWithGenre = 3,
}
type GetAllOptions = PaginationOptions & {
	include?: ContentfulIncludeOptions;
	query?: {
		id?: string;
		title?: string;
	};
};
```

This would also allow same value with different descriptions so we can just keep adding as we need:
```typescript
export enum ContentfulIncludeOptions  {
	movies = 2,
	moviesWithGenre = 3,
	genres = 2,
}

```

Reply:
I like it, small nitpick tho, could we use const instead?

```ts
const ContentfulIncludeOptions = {
	movies: 2,
	moviesWithGenre: 3,
} as const;

type ContentfulIncludeOptions =
	(typeof ContentfulIncludeOptions)[keyof typeof ContentfulIncludeOptions];

type GetAllOptions = PaginationOptions & {
	include?: ContentfulIncludeOptions;
	query?: {
		id?: string;
		title?: string;
	};
};
```

Reply:
Yeah! That's good IMO 👍

Reply:
I concur!

Reply:
Implemented a shared type and const "enum"